commcare_export/commcare_minilinq.py

```diff
@@ -86,8 +43,8 @@ def __call__(self, since, until):
 
 def get_paginator(resource, page_size=1000):
     return {
-        'form': DatePaginator('form', ['server_modified_on','received_on'], page_size),
-        'case': DatePaginator('case', 'server_date_modified', page_size),
+        'form': DatePaginator('form', ['inserted_at','received_on'], page_size),
```

Comment:
Sorry I only noticed this now but I don't think we need to sort by both `inserted_at` and also `received_on`.

Reply:
thanks, fixed in 7f03f7450ca02ea51433843035559fb840a493c3